superset-frontend/src/dashboard/components/gridComponents/Chart.jsx

```diff
@@ -87,7 +87,8 @@ const SHOULD_UPDATE_ON_PROP_CHANGES = Object.keys(propTypes).filter(
 );
 const OVERFLOWABLE_VIZ_TYPES = new Set(['filter_box']);
 const DEFAULT_HEADER_HEIGHT = 22;
-
+// superset backend supports upto a million rows
+const MAXIMUM_NUMBER_OF_ROWS_IN_CSV = 1000000;
```

Comment:
I have a little concern about too many configs in different places. Is possible to read this parameter from backend and carry it to frontend as part of bootstrap data. check this place:
https://github.com/apache/superset/blob/cc2b4fe3f471814fb75c1c8bce97867c96caaec7/superset/views/base.py#L84